app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/Routes.kt

```diff
@@ -12,12 +12,12 @@ object Routes {
     const val Snippets = "snippets"
     const val Profile = "profile"
 
-    fun RepositoryBrowser(data: RepositoryBrowserData) =
+    fun repositoryBrowser(data: RepositoryBrowserData) =
```

Comment:
@BottleRocket-Colin Any input on this one?

Reply:
Since it's a noun and it maintains consistency with the rest of  file, I'd say we should add a suppress notation to Routes for this one and keep it PascalCase.

Reply:
Changed back to PascalCase and added the suppression